subjects/devops/write_file/README.md

```diff
@@ -0,0 +1,86 @@
+## Write file
+
+### Instructions
+
+Create a file `write_file.py` which will have a function `to_do(input)` with one input a `list[tuple[dt.date, str]]` which will be a to do list example:
+
+```python
+[
+    (dt.date(2022, 6, 1), "fix the seat"),
+    (dt.date(2022, 6, 2), "take the trash"),
+]
+```
+
+Your function needs to format and write the input in a text file `output.txt` as follow:
+
+```bash
+$ cat output.txt
+Wednesday 01 June 2022: fix the seat
+Thursday 02 June 2022: take the trash
+$
+```
+
+Make sure you follow the correct format for the output.
```

Comment:
What should happen when the list passed is empty?

What should happen if the format of the list is not the one required? Should the student always assume that the list is correctly formatted?

Reply:
i will add clarification on the readme, but yes they should assume the list is correct.